joomla/joomla-cms
# [4.0] Namespace com_banners

Pull Request for Issue # .

### Summary of Changes
As title says, this PR convert com_banners and it's module to Namespace MVC.

### Testing Instructions
1. Install Joomla 4 (with sample data for easier testing)
2. Apply patch
3. Access to com_banners from backend of the site, try to add/edit/publish/unpublish, delete Banners, Clients, make sure it is working as expected.
4. Edit the **Support Joomla!** banner, look at **Banner Details** tab, set **Track Impressions** and **Track Clicks** parameter to Yes (so that you can test **Tracks** management in later step
5. Access to frontend, click on **Banner** link in **All Modules** module.  There will be a banner displayed, click on it, see that it works
6. Access to com_banners from backend again, try to access to **Tracks** link, see list of Tracks. Click on Export Button and make sure the tracks data is downloaded

Overall, the test is just to make sure it works as how it was (non-namespaced version)
